server/eclipse-project/src/main/java/us/freeandfair/corla/model/CVRContestInfo.java

```diff
@@ -110,14 +110,16 @@ public CVRContestInfo(final Contest the_contest, final String the_comment,
     my_comment = the_comment;
     my_consensus = the_consensus;
     my_choices.addAll(the_choices);
+    boolean isIRV = my_contest.description().equalsIgnoreCase(ContestType.IRV.toString());
+
```

Comment:
What happens when an additional type of contest is considered? With this model there are If IRV then .. in many classes.